bin/cml-cloud-runner-entrypoint.js

```diff
@@ -1,40 +1,40 @@
 #!/usr/bin/env node
 
 const { spawn } = require('child_process');
+
 const { exec, randid } = require('../src/utils');
-const { URL } = require('url');
+const CML = require('../src/cml');
 
 const {
   DOCKER_MACHINE, // DEPRECATED
 
   RUNNER_PATH,
-  RUNNER_REPO,
   RUNNER_IDLE_TIMEOUT = 5 * 60,
   RUNNER_LABELS = 'cml',
   RUNNER_NAME = randid(),
   RUNNER_EXECUTOR = 'shell',
   RUNNER_RUNTIME = '',
-  RUNNER_IMAGE = 'dvcorg/cml:latest'
+  RUNNER_IMAGE = 'dvcorg/cml:latest',
+
+  RUNNER_DRIVER,
```

Comment:
It seems like the flags and some js modules use `driver`, but others use `client`- if I understand correctly, they're referring to the same thing. Is this intentional?

Reply:
I dont remember to have set any client as a parameter, however you are right the ```driver``` points to one of the clients GitlabClient or GitlabClient.
Im changing them  for Gitlab and GitHub and are going to be inside the drivers folder. To be more clear